certificate/endpoint.go

```diff
@@ -67,19 +68,19 @@ func (endpoint Endpoint) Sign(ctx context.Context, req *pb.SigningRequest) (_ *p
 		MinDifficulty: endpoint.minDifficulty,
 	})
 	if err != nil {
-		endpoint.log.Error(zap.Error(err).String)
-		return nil, internalErr(zap.Error(err).String)
+		msg := "error claiming authorization"
+		return nil, endpoint.sanitizer.Error(msg, err)
 	}
 
 	difficulty, err := peerIdent.ID.Difficulty()
 	if err != nil {
-		endpoint.log.Error(zap.Error(err).String)
-		return nil, internalErr(zap.Error(err).String)
+		msg := "error checking difficulty"
+		return nil, endpoint.sanitizer.Error(msg, err)
 	}
 	token, err := authorization.ParseToken(req.AuthToken)
 	if err != nil {
-		endpoint.log.Error(zap.Error(err).String)
-		return nil, internalErr(zap.Error(err).String)
+		msg := "error parsing auth token"
+		return nil, endpoint.sanitizer.Error(msg, err)
```

Comment:
This error isn't internal it's an `INVALID_ARGUMENT` one.

This is was an existing bug so you can leave it for being addressed in a specific one, but open a ticket for not forgetting it to fix it. Comment on the ticket to review the other errors returned by this endpoint, I think that there is some of them that they aren't internal too.

Reply:
This is still happening despite the new sanitizer because the errors retuned by this function aren't of class `authorization.ErrInvalidClaim`